rocky/tools/migrations/0034_update_perms_move_and_clear_groups.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.2.18 on 2023-04-24 13:31
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+from tools.models import OrganizationMember
+
+User = get_user_model()
+
+
+# https://stackoverflow.com/a/40092780/1336275
+def migrate_permissions(apps, schema_editor):
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
```

Comment:
There is no need for this in this migration as far as I can see. Either the admin and redteam group exists, but then we have already run migrate before and the permissions have been created, or the groups don't exist but then we also don't need the permissions.